tests_integration/__tests__/__snapshots__/write.js.snap

```diff
@@ -3,7 +3,7 @@
 exports[`do not write file with --write + formated file (stderr) 1`] = `""`;
 
 exports[`do not write file with --write + formated file (stdout) 1`] = `
-"formated.js%s %dms
+"formated.jsformated.js %dms
```

Comment:
I just realize I removed the `[2K[1G` (remove line) control character in the snapshot output (`strip-ansi`) so that it looks weird here, I think we can add a special case for it to actually do the same thing in the mocked stdout/stderr.